### DISCUSSION TITLE
Full config of MM2

### DISCUSSION THREAD

**Discussion by phuongdpham** [strimzi/strimzi-kafka-operator#9826] [Q&A]
![image](https://github.com/strimzi/strimzi-kafka-operator/assets/12036726/19e138af-d23a-4b0f-b7fd-905feba05a7a)

Can anyone help me where to find full configs of MM2 clusters or mirrors config?

Thanks,

**Comment by scholzj** [comment]
I'm not sure what exactly you mean with _full configs_. The screenshot is from a field in our API. What ever you set there will be in the `KafkaMirrorMAker2` resource you use.
